### ISSUE TITLE
Can't view file called 'help'

### ISSUE THREAD

**Issue by teeberg** [sharkdp/bat#168]
- I tried `bat help`, which gives me the help output
- I tried `bat ./help` which yields:
```
error: The subcommand './help' wasn't recognized
	Did you mean 'help'?

If you believe you received this message in error, try re-running with 'bat -- ./help'
```
- So I tried `bat -- ./help`, which returns the same as above...

Is there another way that I'm not aware of that lets me see that file's contents using `bat`?

It looks to me like `--` doesn't really mean "end of arguments"

**Comment by sharkdp** [comment]
Which version of `bat` are you on? This should have been fixed already (see #82). I can't reproduce this with v0.4.1.

**Comment by teeberg** [comment]
Thank you for your fast response! I should've known to double-check the version, I think I blindly assumed my package manager would've kept it up-to-date, but I must've installed it differently, so I was running an old version: 0.2.3. I just re-installed `bat` through homebrew and now it pulled in 0.4.1 and it seems fixed. Thank you for pointing that out and for the help! 👍🏻
